### PR TITLE
Use clusterName to compute replica node affinity

### PR DESCRIPTION
**Pull request by joshi4**

### CODE REVIEW

**Review by aduong — APPROVED** [review]

File: operator/cluster/cluster_strategy_1.go [hunk 1]
```diff
@@ -21,9 +21,13 @@ package cluster
 import (
 	"bytes"
 	"encoding/json"
+
 	log "github.com/Sirupsen/logrus"
 	crv1 "github.com/crunchydata/postgres-operator/apis/cr/v1"
+
 	//"github.com/crunchydata/postgres-operator/config"
+	"os"
```
Comment (wendorf) [inline operator/cluster/cluster_strategy_1.go:29]: This import should be moved up, and we should probably get rid of that commented-out import.

**Review by wendorf — CHANGES_REQUESTED** [review]
Should this commit be based on `render-master` instead of `master`?

**Comment by aduong** [comment]
Didn't even notice the master v render-master, but yes, it should be render-master.

File: operator/cluster/cluster_strategy_1.go (hunk 1 above)
Comment (joshi4) [reply operator/cluster/cluster_strategy_1.go]: this was resolved when changing the branch from master to render-master.

**Review by wendorf — APPROVED** [review]
🦷

**Comment by joshi4** [comment]
I've tested this operator locally in my dev environment and could create/suspend/resume databases as usual. Going to merge this in now.

**Pull request merged**